hack/update-generated-completions.sh

```diff
@@ -9,6 +9,9 @@ set -o pipefail
 OS_ROOT=$(dirname "${BASH_SOURCE}")/..
 source "${OS_ROOT}/hack/lib/init.sh"
 
+# ensure we have the latest compiled binaries
+"${OS_ROOT}/hack/build-go.sh"
```

Comment:
I'd recommend cleaning up `Godeps/_workspace/pkg` before to make sure the compiled dependencies are also up-to-date. Do either `make clean build` or ` make clean ; hack/build-go.sh`.